policyengine_us/tests/policy/baseline/gov/states/mt/tax/income/deductions/ctcc/mt_child_dependent_care_expense_deduction.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Care expense greater than maximum amount with 1 child
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 1
+    tax_unit_childcare_expenses: 2_401
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 2_400
+  
+- name: Care expense less than maximum amount with 1 child
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 1
+    tax_unit_childcare_expenses: 2_000
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 2_000
+
+- name: Care expense greater than maximum amount with 2 children
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 2
+    tax_unit_childcare_expenses: 3_601
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 3_600
+
+- name: Care expense less than maximum amount with 2 children
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 2
+    tax_unit_childcare_expenses: 3_000
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 3_000
+
+- name: Care expense greater than maximum amount with 3 children
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 3
+    tax_unit_childcare_expenses: 4_801
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 4_800
+
+- name: Care expense less than maximum amount with 3 children
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 3
+    tax_unit_childcare_expenses: 4_000
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 4_000
```

Comment:
please add `mt_agi` to the tests and check if the calculation works as predicted (reduced amount, partial and full)